packages/metal-clay-button/src/MetalClayButton.soy

```diff
@@ -15,13 +16,14 @@
 	{@param? type: string}
 	{@param? value: string}
 
+
 	{if $href}
-		<a href="{$href}" {call .attributes data="all" /}>
+		<a href="{$href}" {call .attributes data="all" /} aria-label="{$aria_description}">
 			{call .icon data="all" /}
 			{$label or ''}
 		</a>
 	{else}
-		<button {call .attributes data="all" /}>
+		<button {call .attributes data="all" /} aria-pressed="false">
```

Comment:
Hey @julien, from what I understood, aria-pressed is only used for toggle buttons, buttons that have two states, like a `checkbox` element. We don't have this type yet, maybe in the future.

https://www.w3.org/TR/wai-aria/states_and_properties#aria-pressed

cc. @pat270